content/workshops/rhel_8/exercise1.0.adoc

```diff
@@ -0,0 +1,236 @@
+
+To get started, we will take a tour of the web console. The web console (called `cockpit`) will be your primary interactive interface to RHEL. The console has a number of functions that allow you to manage most aspects of the installation.
+
+=== Section 1: Overview of the web console
+
+Here is a quick overview of all the available modules:
+
+[.float-group]
+--
+image::cockpit_topics.png[caption="Figure 1: ", title="Web console functions", width=450, float=left]
+The various functional sections can be seen in `Figure 1`, and are: +
+{empty} +
+*System* +
+Contains an overview of the system, including metadata and health graphs
+{empty} +
+*Logs* +
+Displays current `journald` logs from all services
+{empty} +
+*Storage* +
+Displays storage activity graphs, and logs. Allows for the management of NFS mounts, RAID devices, volume groups, VDO (deduplication) devices, and iSCSI targets
+{empty} +
+*Networking* +
+Displays network activity, logs, and allows firewall and network maintenance
+{empty} +
+*Podman Containers* +
+Allows for the management of container images and running containers
+{empty} +
+*Virtual Machines* +
+Virtual machines can be created, deleted and controlled
+{empty} +
+*Accounts* +
+Allows for the creation, deletion and configuration of user accounts and ssh keys
+{empty} +
+*Services* +
+Displays all system targets, services, sockets, timers and paths, as managed by `systemd`, the root process
+{empty} +
+*Session Recording* +
+Manages recorded sessions, and allows for the configuration of session recorder defaults
+{empty} +
+*Applications* +
+Allows for the installation and removal of web console modules
+{empty} +
+*Diagnostic Reports* +
+Creates diagnostic reports used by Red Hat support, also known as SOSreports
+{empty} +
+*Kernel Dump* +
+Manages kernel debugging dumps, created when the system crashes
+{empty} +
+*SELinux* +
+Allows for the setting of SELinux to permissive mode, and shows recent SELinux alerts (`AVCs`)
+{empty} +
+*Software Updates* +
+Manages automatic package updating, and allows for the manual installation of updates
+{empty} +
+*Subscriptions* +
+Manages subscriptions attached to the host
+{empty} +
+*Terminal* +
+Provides an interactive shell
+--
+
+Next, let's proceed to a more detailed examination of each module. Please click through the web console on your host, as you read the explanations, here.
+
+=== Section 2: System Module
+
+image::cockpit_system.png[caption="Figure 2: ", title='Web Console System Module', 500]
+
+The system module is the main screen that you see, when you first login to the web console. It contains an overview of the system, including the below sections:
+
+* hardware type
```

Comment:
Lines 94 - 108 Consider capitalizing the field names. (Hardware Type, Asset Tag, etc.)